src/core/core.scale.js

```diff
@@ -433,7 +454,7 @@ var Scale = Element.extend({
 
 				// Allow 3 pixels x2 padding either side for label readability
 				if (maxLabelWidth + 6 > tickWidth) {
-					tickWidth = maxWidth / (ticks.length - (options.offset ? 0.5 : 1));
+					tickWidth = maxWidth / (ticks.length - options.offset ? 0.5 : 1);
```

Comment:
The addition/subtraction operators have higher precedence than the conditional operator. So, this line should be:
```js
tickWidth = maxWidth / (ticks.length - (options.offset ? 0.5 : 1));
```